ceph.spec.in

```diff
@@ -884,6 +884,8 @@ mkdir -p %{buildroot}%{_sbindir}
 install -m 0644 -D src/logrotate.conf %{buildroot}%{_sysconfdir}/logrotate.d/ceph
 chmod 0644 %{buildroot}%{_docdir}/ceph/sample.ceph.conf
 install -m 0644 -D COPYING %{buildroot}%{_docdir}/ceph/COPYING
+#modify aio-max-nr =1048576
+install -m 0644 -D src/90-cephsysctl.conf %{buildroot}%{_prefix}/lib/sysctl.d/90-cephsysctl.conf
```

Comment:
Can we call this 90-ceph-osd-sysctl.conf, and put it in the ceph-osd package?

Reply:
then it's just a matter of adding a line to debian/ceph-osd.install for usr/lib/sysctl.d/90-ceph-osd-sysctl.conf